mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,23 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
 function addNumbers(a b c) {
-  return a + b + c;
+  return a + b + c;}
+  function addNumbers(a, b, c) {
```

Comment:
Afraid this won't work as you'd think see my notes about the multiply function in exercises/K-functions-parameters/exercise.js If there's anything you don't understand in it let me know